Meta/build-image-qemu.sh

```diff
@@ -99,11 +99,15 @@ if [ -f _disk_image ]; then
     fi
 fi
 
+# source qemy-paths.sh for the SERENITY_QEMU_IMG_BIN variable so qemu-img works
+# correctly on windows
+. "$(dirname "${0}")/qemu-paths.sh"
```

Comment:
Not sure if this helps your shellcheck issue, but we already have SCRIPT_DIR on line 45